src/patternfly/components/BackgroundImage/background-image.scss

```diff
@@ -18,6 +18,11 @@
   background-color: var(--#{$background-image}--BackgroundColor);
   background-image: var(--#{$background-image}--BackgroundImage);
   background-repeat: no-repeat;
-  background-position: var(--#{$background-image}--BackgroundPosition);
   background-size: var(--#{$background-image}--BackgroundSize);
+
+  @include pf-v5-bidirectional-style(
+    $prop: background-position,
+    $ltr-val: var(--#{$background-image}--BackgroundPosition),
+    $rtl-val: bottom left
```

Comment:
I wonder if we should use this on the `--#{$background-image}--BackgroundPosition` declaration instead? This update opts you out of the var to control the background position. On this line https://github.com/patternfly/patternfly/blob/b985207b7f7b861d9b9301d22d3f7d32ee6b2465/src/patternfly/components/BackgroundImage/background-image.scss#L6


we'd update to
```
@include pf-v5-bidirectional-style(
    $prop: --#{$background-image}--BackgroundPosition,
    $ltr-val: bottom right,
    $rtl-val: bottom left
```

wdyt @srambach @thatblindgeye

Reply:
That could make sense. Either way there's always the chance it may not work as intended for a consumer.

- If we leave it as the PR currently is, the background image may be covered by some other element(s) when the p background position is updated
- If we update the var instead of the property, it may prevent the background image from being positioned based on RTL, which could cause it to be covered by other elements that are swapped to the other side of the screen

Updating the var may be the better guesstimate for us to make; at that point it'd be on the consumer to ensure they're customizing the var correctly for LTR and RTL.